tests/input_formats/owasp_deependency_scan/services/test_convert_to_opossum.py

```diff
@@ -0,0 +1,232 @@
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+import json
+
+from packageurl import PackageURL
+
+from opossum_lib.core.entities.external_attribution_source import (
+    ExternalAttributionSource,
+)
+from opossum_lib.core.entities.metadata import Metadata
+from opossum_lib.core.entities.opossum import Opossum
+from opossum_lib.core.entities.opossum_package import OpossumPackage
+from opossum_lib.core.entities.root_resource import RootResource
+from opossum_lib.core.entities.source_info import SourceInfo
+from opossum_lib.input_formats.owasp_deependency_scan.entities.owasp_dependency_report_model import (  # noqa: E501
+    EvidenceCollectedModel,
+    ProjectInfoModel,
+)
+from opossum_lib.input_formats.owasp_deependency_scan.services.convert_to_opossum import (  # noqa: E501
+    convert_to_opossum,
+)
+from tests.setup.owasp_dependency_scan_faker_setup import OwaspFaker
+
+
+class TestConvertMetadata:
+    def test_convert_metadata(self, owasp_faker: OwaspFaker) -> None:
+        owasp_model = owasp_faker.owasp_dependency_report_model()
+        owasp_project_info: ProjectInfoModel = owasp_model.project_info
+
+        opossum: Opossum = convert_to_opossum(owasp_model)
+
+        metadata: Metadata = opossum.scan_results.metadata
```

Comment:
```suggestion
        opossum = convert_to_opossum(owasp_model)

        metadata = opossum.scan_results.metadata
```
these type annotations should be superfluous, or?

Reply:
same for the other occurrences in this file

Reply:
it helped intelliJ a lot